contracts/StoryDataSource.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.9;
+
+import "@chainlink/contracts/src/v0.8/ChainlinkClient.sol";
+import "@chainlink/contracts/src/v0.8/ConfirmedOwner.sol";
+import "./interfaces/IOffChainDataSource.sol";
+import { Strings } from '@openzeppelin/contracts/utils/Strings.sol';
+import "./interfaces/IStoryOracleCallback.sol";
+import "./libraries/StoryOracleClient.sol";
+import "./libraries/StoryOracleHelper.sol";
+
+contract StoryDataSource  is IOffChainDataSource, IStoryOracleCallback, StoryOracleClient {
+    using StoryOracleHelper for StoryOracleHelper.Request;
+    using Strings for uint256;
+
+    mapping(address=>string) public userData;
```

Comment:
Since we have getData function as a getter for userData, do we still need to make userData map public?

Reply:
yep, we can make the userData private.